docs/code-of-conduct.md

```diff
@@ -34,7 +34,7 @@ This Code of Conduct applies both within project spaces and in public spaces whe
 
 ## Enforcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at support@wptide.org. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
+Instances of abusive, harassing or otherwise, unacceptable behavior may be reported by contacting the project team at support@wptide.org. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regards to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```

Comment:
This change is to a file we essentially inherit from https://github.com/ContributorCovenant/contributor_covenant, so you'll want to submit any potential changes upstream there and we will eventually inherit those changes.  In the meantime, please revert this change.